google/cloud/pubsub/testing/mock_message_batch.h

```diff
@@ -16,9 +16,9 @@
 #define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUB_TESTING_MOCK_MESSAGE_BATCH_H
 
 #include "google/cloud/pubsub/internal/message_batch.h"
-#include "google/cloud/pubsub/version.h"
+#include "google/cloud/pubsub/message.h"
+#include "google/cloud/internal/version.h"
```

Comment:
In general we prefer `google/cloud/version.h` (`google/cloud/pubsub/version.h` is for backwards compat).  Can we teach IWYU to assume we prefer `google/cloud/version.h`?

Reply:
This one was my bad. I used `internal/version.h`. I was writing the suggestions from the google3 clang tidy. Clang tidy knows better.... 

Fixed.